src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tab = ({ tab, currentTab, onTabSelected }) => (
+  <li
+    className={cn({ 'is-active': tab.id === currentTab.id })}
+    data-cy="Tab"
+    key={tab.id}
+  >
+    <a
+      href={`#${tab.id}`}
+      data-cy="TabLink"
+      onClick={() => {
+        if (tab.id !== currentTab.id) {
+          onTabSelected(tab);
+        }
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here